guides/common/modules/proc_generating-host-monitoring-reports.adoc

```diff
@@ -7,12 +7,9 @@ To schedule reports, configure a cron job or use the {ProjectWebUI}.
 .Procedure
 . In the {ProjectWebUI}, navigate to *Monitor* > *Reports* > *Report Templates*.
 For example, the following templates are available:
-Host {endash} Installed Products:: Use this template for hosts in Simple Content Access (SCA) organizations.
-It generates a report with installed product information along with other metrics included in *Subscription {endash} Entitlement Report* except information about subscriptions.
+Host {endash} Installed Products:: This generates a report with installed product information along with other metrics included in *Subscription {endash} Entitlement Report* except information about subscriptions.
 
-Subscription {endash} Entitlement Report:: Use this template for hosts that are not in SCA organizations.
-It generates a report with information about subscription entitlements including when they expire.
-It only outputs information for hosts in organizations that do not use SCA.
+Subscription {endash} Entitlement Report:: This generates a report with information about subscription entitlements including when they expire.
```

Comment:
```suggestion
```

I would say remove this. That report will no longer be useful. I will add a task to remove the actual report as well.